modules/oadp-backup-restore-for-large-usage.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// about-installing-oadp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-backup-restore-for-large-usage_{context}"]
+= NodeAgent CPU for large usage
+
+Testing shows that increasing `NodeAgent` CPU can significantly improve backup and restore times when using {oadp-first}.
+
+It is not recommended to use Kopia without limits in production environments on nodes running production workloads due to Kopia’s aggressive consumption of resources.
+
```

Comment:
Wherever you end up putting it, I do think these sentences should be in the same paragraph and not split between lines 
```suggestion
```